IonDotnet/IonType.cs

```diff
@@ -47,7 +72,7 @@ public static class IonTypeExtensions
         /// </summary>
         /// <param name="t">IonType enum</param>
         /// <returns>true when the this is a scalar type</returns>
-        public static bool IsScalar(this IonType t) => t > IonType.None && t < IonType.Clob;
+        public static bool IsScalar(this IonType t) => t.Id > IonType.None.Id && t.Id < IonType.Clob.Id;
```

Comment:
Don't forget to update this logic when IonType.None is removed.

Reply:
Done.